conans/client/cmake.py

```diff
@@ -156,3 +165,48 @@ def runtime(self):
         if runtime:
             return "-DCONAN_LINK_RUNTIME=/%s" % runtime
         return ""
+
+    def configure(self, conan_file, args=None, vars=None, source_dir=None, build_dir=None):
```

Comment:
By the way I just realized it would be nice to have a parameter "parallel=True/False" for the build() method, I've seen examples where people add "-- -j8" which works fine with make but fails for VS.